api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -29,8 +29,8 @@ message RateLimit {
     // X-RateLimit headers disabled.
     OFF = 0;
 
-    // Use `draft RFC Version 02 <https://tools.ietf.org/id/draft-polli-ratelimit-headers-02.html>`_.
-    DRAFT_VERSION_02 = 1;
```

Comment:
So this is just added in the previous PR, right?

Reply:
Yes.